openlibrary/templates/lists/export_as_html.html

```diff
@@ -43,5 +42,39 @@ <h3 class="title">
             <br/>
         </li>
     </ul>
+    <ul>
+    $for book in works:
+        <li>
+            <h3 class="title">
+                <a href="$request.home$book.url()">$book.get('title', 'Title unknown')</a>
+            </h3>
+            by $:render_authors(book)
+
+            <div class="published">
+                $if book.first_publish_year:
+                    $book.first_publish_year
+                $else:
+                    <em>First publish date unknown</em>
+            </div>
+            <br/>
+        </li>
+    </ul>
+    <ul>
+    $for author in authors:
+        <li>
+            <h3 class="title">
+                <a href="$request.home$author.url()">$author.get('name', 'Name unknown')</a>
+            </h3>
+            $if author.birth_date:
+                ( $author.birth_date
+            $else:
+                <em>( - </em>
+            $if author.death_date:
+                - $author.death_date )
+            $else:
+                <em> )</em>
```

Comment:
```suggestion
            $if author.birth_date:
                ( $author.birth_date -
            $else:
                <em>( - </em>
            $if author.death_date:
                $author.death_date )
            $else:
                <em> )</em>
```
Unchanged, this would render two hyphens when the author has a death date but doesn't have a birth date (not sure if this is common, or even possible).